alerta-covid/app/assets/stylesheets/styles.scss

```diff
@@ -0,0 +1,6 @@
+body {
+  background-color: #fff;
+  color: #333;
+  margin: 33px;
+  font-family: verdana, arial, helvetica, sans-serif;
+}
```

Comment:
I'd suggest creating a class instead of styling the actual `body` element 
Some times in rails projects we use different layouts depending on the scope of the application
Defining a global style as we are doing here, will style all `body` elements the same way in all layouts

I know for now we only have the `application` layout, but it's something we should keep in mind